libs/utilities/src/tests/stringUtils.test.ts

```diff
@@ -0,0 +1,71 @@
+// (C) 2007-2020 GoodData Corporation
+import { randomString, shortenText, simplifyText, parseStringToArray, hashCodeString } from "../stringUtils";
+
+describe("#randomString", () => {
```

Comment:
Nitpick: we do not use the `#something` notation anywhere else. Should we remove it here?

Reply:
fixed